pyscriptjs/tests/integration/test_06_element.py

```diff
@@ -0,0 +1,296 @@
+from .support import PyScriptTest
```

Comment:
Just a comment on the name of the file, according to discussions on #995 we decided to remove all order of actions that aren't fundamental to PyScript, meaning the number should go here.